sample/DateTimeParser/DateTimeParser.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="superpower" Version="1.0.2" />
```

Comment:
Hi @Platzer - looks like a few of these are remaining; my guess is this one in particular was blocked on an update to the RTM "dotnet" tooling? If so, I've just done the upgrade in #11, so should be good-to-go now. HTH!